A-test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git

```

Comment:
- [ ] Based on project requirements, this file should be ignored by git. This happens automatically when you include the file name `test.md` into the `.gitignore` file which seems empty. kindly make that happen 👍 

## Instructions :
![image](https://user-images.githubusercontent.com/50990925/219532237-a81161d3-19d0-4058-86de-3fbcf126b7b4.png)